custom_components/sat/coordinator.py

```
@@ -296,17 +299,17 @@ def minimum_relative_modulation_value(self) -> float | None:
     def maximum_relative_modulation_value(self) -> float | None:
         return None
 
+    @property
+    def minimum_setpoint(self) -> float:
+        """Return the minimum setpoint temperature before the device starts to overshoot."""
+        return float(self._data.get(CONF_MINIMUM_SETPOINT))
```

Ruff (0.14.2)

`CONF_MINIMUM_SETPOINT` may be undefined, or defined from star imports

(F405)